doc/documentation_data_format.rst

```diff
@@ -41,12 +41,11 @@ Overview
 ---------
 
 The PEtab data format specifies a parameter estimation problem using a number
-of text-based files (`Systems Biology Markup Language (SBML) <http://sbml.org>`_
-and
+of text-based files (
 `Tab-Separated Values (TSV) <https://www.iana.org/assignments/media-types/text/tab-separated-values>`_)
 (Figure 2), i.e.
 
-- An SBML model [SBML]
+- An model
```

Comment:
```suggestion
- A model
```

Reply:
Could define currently supported formats here (e.g. the "Initial set of model format identifiers")